app/controllers/admin/users_controller.rb

```diff
@@ -3,9 +3,18 @@ class Admin::UsersController < ApplicationController
   before_action :check_admin
 
   def index
-    @users = User.all
+    if params[:waiting_approval]
+      @users = User.where.not(approved: true)
```

Comment:
it is better to have such logic as a `scope` => `User.waiting_approval`

http://api.rubyonrails.org/classes/ActiveRecord/Scoping/Named/ClassMethods.html#method-i-scope